packages/expo-video-thumbnails/android/src/main/java/expo/modules/videothumbnails/VideoThumbnailsModule.java

```diff
@@ -0,0 +1,88 @@
+package expo.modules.videothumbnails;
+
+import java.util.Map;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.OutputStream;
+import java.io.IOException;
+import java.util.UUID;
+
+import android.content.Context;
+import android.media.MediaMetadataRetriever;
+import android.os.Bundle;
+import android.graphics.Bitmap;
+import android.net.Uri;
+
+import org.unimodules.core.ExportedModule;
+import org.unimodules.core.ModuleRegistry;
+import org.unimodules.core.Promise;
+import org.unimodules.core.interfaces.ExpoMethod;
+import org.unimodules.core.interfaces.ModuleRegistryConsumer;
+import expo.modules.videothumbnails.Options;
+
+public class VideoThumbnailsModule extends ExportedModule implements ModuleRegistryConsumer {
```

Comment:
In general modules don't need to implement `ModuleRegistryConsumer` — you only have to implement it if you later use it. 🙂

However, in this case you correctly implemented it since you should use file system permissions interface to ensure that a module should be given access to requested URL